server/azureAppInsights.ts

```diff
@@ -4,14 +4,20 @@ import applicationVersion from './applicationVersion'
 import config from './config'
 import logger from '../log'
 
-function ignoreStaticAssetsProcessor(
+function ignoreExcludedRequestsProcessor(
   envelope: Contracts.EnvelopeTelemetry,
   _contextObjects: { [name: string]: unknown } | undefined
 ): boolean {
   if (envelope.data.baseType === Contracts.TelemetryTypeString.Request) {
     const requestData = envelope.data.baseData
-    if (requestData instanceof Contracts.RequestData && requestData.name.startsWith('GET /assets/')) {
-      return false
+    if (requestData instanceof Contracts.RequestData) {
+      const { excludedRequests } = config.applicationInsights
+      for (let i = 0; i < excludedRequests.length; i += 1) {
```

Comment:
This doesn't really matter, and I might be missing something or could this be an ES6 `for...of`? Is the index important?

Reply:
intellij really doesn't like you using that thing... the reasons are compelling but boring